webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -701,17 +701,17 @@ <h2>Crashing Thread ({{ crashing_thread }}){% if parsed_dump.threads[crashing_th
                     </thead>
                     <tbody>
                       {% for frame in parsed_dump.threads[crashing_thread].frames %}
-                        <tr class="{% if frame.missing_symbols %}missingsymbols{% endif %}">
-                          {% if frame.truncated %}
-                            <td colspan="5">truncated {{ frame.truncated|digitgroupseparator }} frames...</td>
-                          {% else %}
-                            <td>
+                        {% if frame.truncated %}
+                          <tr><td colspan="5">truncated {{ frame.truncated|digitgroupseparator }} frames...</td></tr>
+                        {% else %}
+                          <tr class="{% if frame.missing_symbols %}missingsymbols{% endif %}">
+                            <td rowspan="{{ rowspan_for_inlines(frame) }}">
```

Comment:
We want to show the function plus the inlines. So the rowspan is 1 + len(inlines) which is easier to do as a function than to do in the template.